文件名下划线后不足三位数字提示：panic: runtime error: invalid memory address or nil pointer dereference

操作系统： Windows 10 x64
Go版本： 1.14.1
AVMeta版本： v1.0.1
配置信息：自动生成的默认配置
错误信息：
文件: [010915_01.mp4] 刮削失败, 错误原因: panic: runtime error: invalid memory address or nil pointer dereference

复现过程：
010915_01.mp4手动补足010915_001.mp4则不会报错

```
D:\test>DIR
 驱动器 D 中的卷没有标签。
 卷的序列号是 2453-EA28

 D:\test 的目录

2020/06/17  23:25    <DIR>          .
2020/06/17  23:25    <DIR>          ..
2019/10/28  22:51     1,073,741,824 010915_01.mp4
2020/03/29  21:40        11,090,944 AVMeta.exe
               2 个文件  1,084,832,768 字节
               2 个目录 318,827,421,696 可用字节

D:\test>AVMeta.exe

共探索到 1 个视频文件, 开始刮削整理...

panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x8cb8ff]

goroutine 6 [running]:
github.com/ylqjgm/AVMeta/pkg/scraper.(*DMMScraper).GetActors(0xc000054c30, 0xc0000021e0)
        /home/runner/work/AVMeta/AVMeta/pkg/scraper/dmm.go:186 +0x3f
github.com/ylqjgm/AVMeta/pkg/media.ParseMedia(0xaba420, 0xc000054c30, 0x0, 0x0, 0x0)
        /home/runner/work/AVMeta/AVMeta/pkg/media/media.go:72 +0x79
github.com/ylqjgm/AVMeta/pkg/media.search(0xc000195380, 0x15, 0xc00019a000, 0x0, 0x0, 0x0)
        /home/runner/work/AVMeta/AVMeta/pkg/media/pack.go:262 +0xa0d
github.com/ylqjgm/AVMeta/pkg/media.capture(0xc000195380, 0x15, 0xc00019a000, 0x0, 0x0, 0x0)
        /home/runner/work/AVMeta/AVMeta/pkg/media/pack.go:120 +0x61
github.com/ylqjgm/AVMeta/pkg/media.packNfo(0xc000195380, 0x15, 0xc00019a000, 0x0, 0x0, 0x0)
        /home/runner/work/AVMeta/AVMeta/pkg/media/pack.go:43 +0x61
github.com/ylqjgm/AVMeta/pkg/media.Pack(0xc000195380, 0x15, 0xc00019a000, 0x0, 0x0, 0x0)
        /home/runner/work/AVMeta/AVMeta/pkg/media/pack.go:33 +0x65
github.com/ylqjgm/AVMeta/pkg/cmd.(*Executor).packProcess(0xc000052660, 0xc000195380, 0x15, 0xc0001b0240)
        /home/runner/work/AVMeta/AVMeta/pkg/cmd/root.go:82 +0x65
created by github.com/ylqjgm/AVMeta/pkg/cmd.(*Executor).rootRunFunc
        /home/runner/work/AVMeta/AVMeta/pkg/cmd/root.go:72 +0x260

D:\test>ren 010915_01.mp4 010915_001.mp4

D:\test>dir
 驱动器 D 中的卷没有标签。
 卷的序列号是 2453-EA28

 D:\test 的目录

2020/06/17  23:27    <DIR>          .
2020/06/17  23:27    <DIR>          ..
2019/10/28  22:51     1,073,741,824 010915_001.mp4
2020/03/29  21:40        11,090,944 AVMeta.exe
2020/06/17  23:26               382 config.yaml
               3 个文件  1,084,833,150 字节
               2 个目录 318,827,421,696 可用字节

D:\test>AVMeta.exe

共探索到 1 个视频文件, 开始刮削整理...

文件: [D:\test\010915_001.mp4] 刮削失败, 错误原因: 404 Not Found

D:\test>AVMeta.exe version
Version:        1.0.1
Git commit:     34daaf4
Built:          2020-03-29T13:38:59Z
Go Version:     go1.14.1
Platform:       windows/amd64
```

--- a/pkg/media/pack.go
+++ b/pkg/media/pack.go
@@ -230,10 +230,10 @@ func search(file string, cfg *util.ConfigStruct) (*Media, error) {
 
 	// 查找正则匹配
 	for _, scr := range sr {
-		// 刮削赋值
-		s = scr.S
 		// 检查是否匹配
 		if scr.R.MatchString(code) {
+			// 刮削赋值
+			s = scr.S
 			// 刮削
 			err = s.Fetch(code)
 			break

